### ISSUE TITLE
VisualScript. Pool Array functions without sequence port

### ISSUE THREAD

**Issue by MajesticGames** [godotengine/godot#38435]
**Godot version:**
3.2.1 Stable

**OS/device including version:**
Windows 10 64 bit

**Issue description:**

I think there's a problem in visual script with pool array variables. If you want to remove, add, insert, values to an existing pool array variable, and you add the functions "Pool XXXXXX array remove", "Pool XXXXXX array append", etc, you will notice that there is no sequence port and no output data port.

![Poolvector](https://user-images.githubusercontent.com/33732701/80916795-4527d480-8d5b-11ea-9405-4b3e62cec782.png)

**Comment by DavidCambre** [comment]
Yes just checked and every pool... base_type has no sequence port.

Isseu #48127 is also dealing with VisualScriptPropertySet Class (Do not know if thay are related)
